orion-ui/src/pages/WorkerPoolCreate.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
Love how simple this component is now that the form component can handle its own data saving and redirects.